projects/packages/search/src/dashboard/components/record-meter/lib/record-info.js

```diff
@@ -6,17 +6,14 @@ const PALETTE = require( '@automattic/color-studio' );
 /**
  * converts provided information into a chart consumable data form
  *
- * @param {object} data - contains record type breakdown, post count, last indexed date,
- * @param {object} planInfo - information about the users current plan
+ * @param {object} data - contains record type breakdown, post count, last indexed date, and tierMaximumRecords
  * @returns {object} data in correct form to use in chart and notice-box
  */
-export default function getRecordInfo( data, planInfo ) {
-	// set max number of record types to display
-	const maxRecordCount = 5;
-
-	// sets up some basic counts and arrays
+export default function getRecordInfo( data ) {
```

Comment:
`data` is very generic and doesn't give us much information at a glance when we look at the function signature.

As these inputs should stay pretty consistent, I would be tempted to provide these as separate arguments to the function (or at least destructure them like we would with component props).